### PR TITLE
Make it possible to load theme/<themename>/dist

### PR DESCRIPTION
**Pull request by CarlSchwan**
This fix a regression with the theming api following the change in
webpack bundling introduced in 24

Signed-off-by: Carl Schwan <carl@carlschwan.eu>

### CODE REVIEW

**Review by artonge — APPROVED** [review]
Sounds right

File: lib/private/Template/JSResourceLocator.php [hunk 1]
```diff
@@ -71,12 +71,14 @@ public function doFind($script) {
...
 			|| $this->appendIfExist($this->serverroot, $script.'.js')
+			|| $this->appendIfExist($this->serverroot, "$theme_dir.'dist/$app-$scriptName.js")
 			|| $this->appendIfExist($this->serverroot, "dist/$app-$scriptName.js")
 			|| $this->appendIfExist($this->serverroot, 'apps/'.$script.'.js')
 			|| $this->cacheAndAppendCombineJsonIfExist($this->serverroot, $script.'.json')
 			|| $this->appendIfExist($this->serverroot, $theme_dir.'core/'.$script.'.js')
 			|| $this->appendIfExist($this->serverroot, 'core/'.$script.'.js')
-			|| (strpos($scriptName, '/') === -1 && $this->appendIfExist($this->serverroot, "dist/core-$scriptName.js"))
+			|| (strpos($scriptName, '/') === -1 && ($this->appendIfExist($this->serverroot, "dist/core-$scriptName.js")
+				|| $this->appendIfExist($this->serverroot, $theme_dir . "dist/core-$scriptName.js")))
```
Comment (CarlSchwan) [inline lib/private/Template/JSResourceLocator.php:81]:
```suggestion
			|| (strpos($scriptName, '/') === -1 && ($this->appendIfExist($this->serverroot, $theme_dir . "dist/core-$scriptName.js")
				|| $this->appendIfExist($this->serverroot, "dist/core-$scriptName.js")))
```

**Review by skjnldsv — APPROVED** [review]

**Comment by CarlSchwan** [comment]
Tested and this is working fine

**Pull request merged**